### ISSUE TITLE
2020 Feb 14: Global Team Call agenda

### ISSUE THREAD

**Issue by JMParr** [programminghistorian/jekyll#1655]
If you are reporting a problem with a lesson, please try to provide us with the following information:

- The full title of the lesson
- The system you are using (Mac, Linux, Windows)
- Version numbers of the relevant software you are using
- The exact steps you took that caused the problem

If you'd like, you can delete this template text before posting your issue.

**Comment by JMParr** [comment]
The editing button doesn't seem to be working. I'll add the agenda here:

- [x] Update on current activities.

- [x] Discuss outreach, and set sustainable priorities.

- [x] Determine how best we can work with the new Outreach Manager (who will be joining us)

- [x] Award opportunity mentioned by @drjwbaker 

- [x] @rivaquiroga will discuss lessons from the Bogota experience.

- [x] @spapastamkou will discuss her work with Senegalese contacts.

**Comment by rivaquiroga** [comment]
I would like to share on Friday some lessons we've learned from the Bogotá experience about the follow-up of participants that may be important for future PH workshops.

**Comment by JMParr** [comment]
Here are the meeting notes from our meeting 2020-02-14:

- We went over our updates, and introduced Penny (our PH Outreach/Impact Manager) to the Global Team. @spapastamkou has been making good contacts in Senegal, and mentioned that the timing is good for Open Science in France. She's going to continue these conversations. @acrymble has been in touch with the proposed Portuguese team about bringing them on board. 

- @rivaquiroga mentioned that for future workshops, we might consider having participants come with a more developed plan, and both @rivaquiroga & @acrymble suggested that it would be useful to have members of the PH team who are not planners/instructors help mentor participants to boost the number of tutorials we get out of these conferences. It was also suggested that we ask for a back-up email of participants.

- @JMParr is going to create a Google doc for the team to work together on drafting publication guidelines to help with communication of expectations for new language teams. @JMParr will also follow up with AMICAL, and QD, and make some introductions with Penny.

- We discussed the need to pace ourselves with new publications, but that our next PH publication should be non-European, and that Arabic is a good goal. 

- We discussed the feasibility of applying for the award that @drjwbaker mentioned. Discussion to be continued early next week.
